compose/flask/start

```diff
@@ -4,5 +4,6 @@ set -o errexit
 set -o pipefail
 set -o nounset
 
+flask run --host=0.0.0.0 --port=5000
 #flask db upgrade
-flask run --host=0.0.0.0 --port=5000
\ No newline at end of file
+#python manage.py insert_data
```

Comment:
I recalled the [cv-model-serve](https://github.com/blancadesal/cv-model-serve) code you'd shown me before and copied a lot of that. I was able to get things running using /start but the other commands here in the start file had no effect. (They're just commented out here because they weren't doing anything and I wanted to commit a "working" version.)